stencils/pace/stencils/testing/translate_physics.py

```diff
@@ -55,52 +51,43 @@ def read_microphysics_serialized_data(self, serializer, savepoint, variable):
             return data
         return rearranged
 
-    def read_dycore_serialized_data(self, serializer, savepoint, variable):
-        data = serializer.read(variable, savepoint)
-        if len(data.flatten()) == 1:
-            return data[0]
-        return data
-
-    def read_dwind_serialized_data(self, serializer, savepoint, varname):
+    def transform_dwind_serialized_data(self, data):
```

Comment:
I'm wondering if we could reuse the other `transform_dwind_serialized_data` in translate_fv_update_phys.py? It was really hard to get things in the right places, so we originally had things duplicated everywhere.

Reply:
Yeah, I can clean this up. I tried to avoid refactors where possible to limit scope, and because I could easily go overboard, but de-duplication is pretty high priority.

Reply:
Thanks for doing this.